docs/releases/3.0.txt

```diff
@@ -392,8 +392,14 @@ backends.
   :class:`~django.db.models.DateTimeField` in ``datetime_cast_date_sql()``,
   ``datetime_extract_sql()``, etc.
 
-* ``DatabaseOperations.return_insert_id()`` now requires an additional
-  ``field`` argument with the model field.
+* ``DatabaseOperations.fetch_returned_insert_id()`` is renamed to
+  ``fetch_returned_insert_columns()`` and returns a list instead of single
+  value.
```

Comment:
```
… is replaced by … which returns a list of values generated by the query during execution instead of a single value, which was typically the auto-incremented primary key.
```